components/shared/StandardStyles.js

```diff
@@ -40,16 +40,12 @@ export const StyledP = styled.p`
   `};
 `;
 
-export const StyledPre = styled.pre`
-  padding-right: 1rem;
-  margin-top: 0;
-  font-weight: 200;
-  line-height: 1.6;
-  font-size: 1.5rem;
+export const StyledP = styled.p`
+  ${ParagraphAndPreSharedStyles}
+`;
 
-  ${below.med`
-    margin-top: 0;
-  `};
+export const StyledPre = styled.pre`
```

Comment:
Looks like this component is used on `JobListing` and `SlimSession` (which I want to say was used in voting). It looks like it for sure needs the font-size there, or included in this component. I believe font-weight too but not sure. And, you could include the height: auto here too actually.

Reply:
@saragibby I tested both of those and noted in the PR.  Not sure why they would need to change as it is the exact same code I just extracted in to a shared style.